src/components/MySearching/index.tsx

```diff
@@ -0,0 +1,10 @@
+interface  serachInterface {
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
don't use such namings like e, for errors name it error, for event name it event